coverage/test/commons.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"testing"
+)
+
```

Comment:
I could only find 1 usage of `AssertEqual`, and no use of the other methods. I suggest simply inline the `AssertEqual` comparison and remove this file.

Reply:
I believe this one would be used more in latter stages. I think it is costly to go back and forth between stages

Reply:
Then just keep `AssertEqual` and remove the rest. It'll make the code simpler and adding the methods later is easy.

Reply:
Done